packages/applications-service-api/src/utils/advice.mapper.js

```diff
@@ -33,7 +33,8 @@ const mapCommonAdviceFieldsToApi = (advice) => {
 		dateEnquiryReceived: advice?.enquiryDate,
 		dateAdviceGiven: advice?.adviceDate,
 		dateLastModified: advice?.modifiedAt,
-		dateCreated: advice?.createdAt
+		dateCreated: advice?.createdAt,
+		title: advice?.title
```

Comment:
We should probably update swagger spec:
.../packages/applications-service-api/api/openapi.yaml

Reply:
https://github.com/Planning-Inspectorate/applications-service/pull/999/commits/4aba185874c6bd78c72ccc127f1dc0e5d8256761